divisiondetector/datamodules.py

```diff
@@ -5,20 +5,43 @@
 
 class DivisionDataModule(pl.LightningDataModule):
 
-    def __init__(self,
-                 batch_size,
-                 loader_workers=10):
-
+    def __init__(self, batch_size, loader_workers=10):
         super().__init__()
+
         self.batch_size = batch_size
         self.loader_workers = loader_workers
 
     def setup(self, stage=None):
+        window_size = (500, 500, 500)
```

Comment:
These should all become parameters of the init function so that we can adjust them later during different training runs.

For example, make window_size an argument of __init__, assign self.window_size = window_size, use self.window_size in setup(). Also, add a commandline argument for each parameter